### ISSUE TITLE
[Feature Request] Fixed combinations

### ISSUE THREAD

**Issue by vanceza** [Chicken-Bones/EnderStorage#52]
Requesting a config file option to disable changing the combination colors of ender chests and tanks.

Use case: chests aren't craftable, they're quest rewards.

**Comment by Ecu** [comment]
I suppose I approve this request as a config option.  I can definitely see the use-case.

**Comment by vanceza** [comment]
Perfect.

I'd suggest locking should prevent changing from those frequencies as well as to them, so people don't get "locked out".

**Comment by Chicken-Bones** [comment]
Do you need the ability to lock specific frequencies?

**Comment by Chicken-Bones** [comment]
Also, consider instead the possibility of only filling the chest with the
quest rewards on completion via a redstone mechanism

**Comment by Ecu** [comment]
I think the ability to lock frequencies can be really useful, @Chicken-Bones.

**Comment by vanceza** [comment]
No, it's not required for me to lock specific frequencies. It would be a small bonus because if I don't lock all frequencies, then players can use the standard ender chest / ender tank functionality in their builds.

Redstone's a good idea. Here are the issues:
- It can't support tracking multiple parties completing quests independently, and giving ender chests as reward can.
- How can we activate a remote redstone mechanism only when a quest is complete? HQM doesn't support redstone signals, only item rewards. I'll look into your wireless redstone mod which I haven't used, and see if I can gimmick a "remote control" reward, this may be solvable.

**Comment by Chicken-Bones** [comment]
Here's an idea: Make your reward chests personal with a special character
name and then give out pouches

**Comment by vanceza** [comment]
I'm not actually giving a one-time gift, but a continuous supply from a factory somewhere far off (which is why I'm using an ender chest instead of just giving them stuff directly).

I know the original ender chests didn't support automation or stay linked to a specific character. I didn't realize your new ender pouches could stay linked to a personal chest, that's really neat. Can personal chests/tanks stay associated with one person and hook up to hoppers or pipes? If so, that's a valid workaround.

[Will investigate on my own if you don't reply, nothing needs done]

**Comment by Ecu** [comment]
~prods @Chicken-Bones~

While that is likely a valid work-around.  I think in this case, the ability to lock frequencies does have valid use-cases.  I really think it should be considered for addition.
